models/manifest.go

```diff
@@ -20,6 +20,14 @@ type ECManifestResult struct {
 
 }
 
+type ECResult struct {
+	ECManifest
+	HostExec		string `json:"host"`
+	StdOutput		[]string `json:"stdOutput"`
+	StdErrOutput	[]string `json:"stdErrOutput"`
```

Comment:
running go fmt should help with formatting.

Reply:
It probably would.  My suspect would be IDE space setting. Will test it out to confirm.